client/my-sites/domains/domain-management/add-dns-record/a-record.jsx

```diff
@@ -0,0 +1,77 @@
+import classnames from 'classnames';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormInputValidation from 'calypso/components/forms/form-input-validation';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormTextInput from 'calypso/components/forms/form-text-input';
+import FormTextInputWithAffixes from 'calypso/components/forms/form-text-input-with-affixes';
+
+class ARecord extends Component {
+	static propTypes = {
+		fieldValues: PropTypes.object.isRequired,
+		onChange: PropTypes.func.isRequired,
+		selectedDomainName: PropTypes.string.isRequired,
+		show: PropTypes.bool.isRequired,
+	};
+
+	render() {
+		const { fieldValues, isValid, onChange, selectedDomainName, show, translate } = this.props;
+		const classes = classnames( { 'is-hidden': ! show } );
+		const isNameValid = isValid( 'name' );
+		const isDataValid = isValid( 'data' );
+		const isAaaaRecord = fieldValues.type === 'AAAA';
+
+		let namePlaceholder = translate( 'Enter subdomain', {
+			context: 'Placeholder shown when entering the optional subdomain part of a new DNS record',
+		} );
+		let dataPlaceholder = translate( 'e.g. %(example)s', {
+			args: {
+				example: '123.45.78.9',
+			},
+		} );
+
+		if ( isAaaaRecord ) {
+			namePlaceholder = translate( 'Enter subdomain (required)', {
+				context: 'Placeholder shown when entering the required subdomain part of a new DNS record',
+			} );
+			dataPlaceholder = translate( 'e.g. %(example)s', {
+				args: {
+					example: '2001:500:84::b',
+				},
+			} );
+		}
+
+		return (
+			<div className={ classes }>
+				<FormFieldset>
+					<FormLabel>{ translate( 'Name (optional)', { context: 'DNS Record' } ) }</FormLabel>
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.